File: web/src/models/siteSummary.ts
import { SiteType } from '@/enums'

export interface SiteSummary {
  id: number
  siteName: string
  siteType: SiteType
  iconUrl: string
  siteDir: string
  description: string
  tableName: string
  root: boolean
  disabled: boolean
  taxis: number

  isMoving: boolean
  isDragOver: boolean
}

File: core/src/SSRAG/Models/SiteSummary.cs
using SSRAG.Enums;

namespace SSRAG.Models
{
    public class SiteSummary
    {
        public int Id { get; set; }
        public string SiteName { get; set; }
        public SiteType SiteType { get; set; }
        public string IconUrl { get; set; }
        public string SiteDir { get; set; }
        public string Description { get; set; }
        public string TableName { get; set; }
        public bool Root { get; set; }
        public bool Disabled { get; set; }
        public int Taxis { get; set; }
    }
}
